<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>serial1.ico</normaloff>serial1.ico</iconset>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>63</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>BaudRate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>67</y>
      <width>63</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>DataBits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>95</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>132</y>
      <width>63</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>StopBits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>33</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Devices:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_BaudRate">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>0</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>57600</string>
    </property>
    <property name="currentIndex">
     <number>8</number>
    </property>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>921600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3000000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4000000</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cB_Devices">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>31</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_DataBits">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>61</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cB_Parity">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>94</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Space</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mark</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cB_StopBits">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>127</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="tE_RecData">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>361</width>
      <height>160</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="tE_SendData">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>176</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_Send">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>346</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_Open">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>161</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_Clear">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>229</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_Save">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>265</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_OpenFile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>346</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Send">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>350</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Rec">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>160</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(237, 237, 237)</string>
    </property>
    <property name="text">
     <string>Hex</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>355</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>355</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_RxNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>356</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_TxNumber">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>356</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_RTClear">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>356</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../../usr/share/icons/default.kde4/16x16/actions/edit-delete.png</normaloff>../../../../../../usr/share/icons/default.kde4/16x16/actions/edit-delete.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_Stop">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>194</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>381</width>
      <height>341</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>381</width>
      <height>341</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>铁路车号</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 232, 191);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
